src/index.ts

```diff
@@ -1,6 +1,18 @@
-export * from './components';
-export * from './constants';
-export * from './containers';
-export * from './helpers';
-export * from './models';
-export * from './store';
+import {useMixer} from './hooks/useMixer';
```

Comment:
from my perspective, there should be only one available thing — the Mixdesk component itself. My suggestion is to cut off the mixdesk itself from /models into a standalone library if you would like to keep things accessible atomically

Reply:
changes on this file reverted and updated with Mixdesk component (take in account, here are some name collision because of reexporting everything on the same level)